.github/workflows/coding-standards.yml

```diff
@@ -24,15 +24,10 @@ jobs:
           php-version: "${{ matrix.php-version }}"
           tools: "cs2pr"
 
-      - name: "Cache dependencies installed with Composer"
-        uses: "actions/cache@v2"
-        with:
-          path: "~/.composer/cache"
-          key: "php-${{ matrix.php-version }}-composer-locked-${{ hashFiles('composer.lock') }}"
-          restore-keys: "php-${{ matrix.php-version }}-composer-locked-"
-
       - name: "Install dependencies with Composer"
-        run: "composer install --no-interaction --no-progress --no-suggest"
+        uses: "ramsey/composer-install@v1"
+        with:
+          composer-options: "--no-interaction --no-progress --no-suggest"
```

Comment:
In fact, for this workflow, I think you can reuse the one from the `.github` repository :slightly_smiling_face: